vendor/engines/bulk_email/app/views/bulk_email/bulk_email/create.html.haml

```diff
@@ -0,0 +1,29 @@
+= render "shared/transactions/headers"
+= stylesheet_link_tag "bulk_email/application.css"
+
+= content_for :h1 do
+  = current_facility
+
+%h2= t("bulk_email.title")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_users", sidenav_tab: "email"
+
+= simple_form_for @delivery_form, method: :post, url: facility_bulk_email_deliver_path do |f|
+  - @users.each do |user|
+    = hidden_field_tag "bulk_email_delivery_form[recipient_ids][]", user.id, id: nil
+
+  .span8
+    = label_tag text("bulk_email.recipients")
+    .well= @users.map(&:email).join(", ")
+
+  .span8= f.input :subject, input_html: { class: "input-xxlarge" }
+
+  .span8
+    = f.input :custom_message,
+      as: :text,
+      input_html: { class: "input-xxlarge", rows: 6 }
+
+  .span8
+    = link_to text("shared.cancel"), facility_bulk_email_path, class: "btn"
```

Comment:
I think our usual styling for "Cancel" is just a link and to the right of the submit button.